.github/workflows/OAS-release.yml

```
@@ -0,0 +1,35 @@
+name: Create OAS release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  OAS-release:
```

```suggestion
name: Create GitHub Release

on:
  workflow_dispatch:

jobs:
  github-release:
```

How do you feel about this? I think it'd be good to be explicit about what it's creating. If you're okay with it we'll need to update the file name also.